sdk/monitor/azure-monitor-query/tests/perfstress_tests/batch_query.py

```diff
@@ -0,0 +1,81 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import asyncio
+from datetime import date, datetime, timezone
+from azure_devtools.perfstress_tests import PerfStressTest
+
+from azure.monitor.query import LogsQueryClient as SyncLogsQueryClient, LogsQueryRequest
+from azure.monitor.query.aio import LogsQueryClient as AsyncLogsQueryClient
+
+from azure.identity import DefaultAzureCredential as SyncDefaultAzureCredential
+from azure.identity.aio import DefaultAzureCredential as AsyncDefaultAzureCredential
+
+class LogsBatchPerfTest(PerfStressTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+
+        # auth configuration
+        self.workspace_id = self.get_from_env('LOG_WORKSPACE_ID')
+
+        # Create clients
+        self.logs_client = SyncLogsQueryClient(
+            credential=SyncDefaultAzureCredential()
+        )
+        self.async_logs_client = AsyncLogsQueryClient(
+            credential=AsyncDefaultAzureCredential()
+        )
+
+        self.requests = [
+            LogsQueryRequest(
+                query="AzureActivity | summarize count()",
+                start_time=datetime(2021, 7, 25, 0, 0, 0, tzinfo=timezone.utc),
+                end_time=datetime.now(),
```

Comment:
Will this test become longer and longer by design, since the time range is increasing every day? This won't be a good indicator of perf regression if the test is written in a way that it can only get slower.

Reply:
```suggestion
                end_time=datetime(2021, 7, 26, 0, 0, 0, tzinfo=timezone.utc),
```